server/models/bhima.sql

```diff
@@ -117,8 +117,12 @@ INSERT INTO unit VALUES
   (229, 'Visits Registry', 'TREE.VISITS_REGISTRY', 'Visits registry', 12, '/modules/patient/visits', '/patients/visits'),
```

Comment:
@mbayopanda  Please add all changes and additions of the database to the migrate file in order to facilitate the deployments of its various modules in the development sites